iroh-sync/src/store.rs

```diff
@@ -16,9 +16,7 @@ use crate::{
     AuthorId, Capability, CapabilityKind, Entry, NamespaceId, PeerIdBytes,
 };
 
-#[cfg(feature = "fs-store")]
 pub mod fs;
```

Comment:
Should we rename this, `fs` seems like the wrong name? If we drop the trait we could export it just as `Store`

Reply:
Not sure about dropping the trait, I think in the longer term we do want a trait here. Still unclear whether dropping the trait is a temp thing to refactor faster or a permanent thing...